packages/core/src/config.ts

```diff
@@ -565,6 +578,14 @@ const composeExternalsConfig = (
           },
         },
       };
+    case 'mf':
```

Comment:
What the error message would be if `@module-federation/rsbuild-plugin` is not installed in userland? Should we precheck the existence of `@module-federation/rsbuild-plugin` to give a clear instruction to user?

Reply:
The current state is to allow users to manually inject`@module-federation/rsbuild-plugin`. Once the plugin is stable, we can consider make it built-in through optional peerDeps so that we can do dependency existence verification during dynamic import.

Reply:
We could check this by checking plugin's name https://github.com/module-federation/core/pull/3062/files#diff-5ee5efd88d835533c55cc15b806482c0b87c4c979df54ce195f0c32063cf43a5R17

Reply:
> What the error message would be if @module-federation/rsbuild-plugin is not installed in userland?

Not only installed, but also should be added into the plugin lists. So by expecting the plugin's name would be better.

Reply:
https://github.com/web-infra-dev/rspress/blob/de549f484dc764fa617dd305350253c8a07414a9/packages/core/src/node/initRsbuild.ts#L43-L50 can copy this

Reply:
added
<img width="812" alt="image" src="https://github.com/user-attachments/assets/03a52e3e-cb60-4217-a353-ee72a80ac2d6">